apps/products/product_test.py

```diff
@@ -61,10 +60,9 @@ def create(self, docs, **kwargs):
         doc["_items"] = results
         return [article_id]
 
-    def test_products(self, article, lookup=None):
-        req = ParsedRequest()
+    def test_products(self, article):
```

Comment:
Planning module will also need to be updated (to remove `lookup=None`): https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/output_formatters/utils.py#L42

Reply:
I see, will keep that param for then to make it easier to backport it